workinggroups/networking.md

```diff
@@ -0,0 +1,53 @@
+# CNCF Networking WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore cloud native networking technology and concepts around the container networking interface (CNI).
```

Comment:
## Objective

Explore cloud native networking technology, concepts around the container networking interface (connectivity), network services, common design patterns and best practices.